label_sync/cluster/label_sync_job.yaml

```diff
@@ -26,12 +26,13 @@ spec:
       restartPolicy: Never  # https://github.com/kubernetes/kubernetes/issues/54870
       containers:
       - name: label-sync
-        image: gcr.io/k8s-testimages/label_sync:v20171215-9c0f9501e
+        image: gcr.io/k8s-testimages/label_sync:v20171219-22a3ef0f6
         args:
         - --config=/etc/config/labels.yaml
         - --dry-run=false
         - --orgs=kubernetes
         - --token=/etc/github/oauth
+        - --only=kubernetes/contrib
```

Comment:
why this change?

Reply:
I've been using it as a canary job before the cronjob mutates all repos. Good way to discover problems

Reply:
maybe add a comment somewhere indicating this is a canary job? it's not very obvious otherwise.

Reply:
meh

Reply:
as this is it looks like a bug. I would likely look at this later and go "weird, this should probably be updating everything" and remove the line.